operator-pipeline-images/operatorcert/entrypoints/index.py

```diff
@@ -124,7 +124,8 @@ def publish_bundle(
     resp = iib.add_builds(iib_url, payload)
 
     batch_id = resp[0]["batch"]
-    response = wait_for_results(iib_url, batch_id)
+    delay = 3600  #60*60
+    response = wait_for_results(iib_url, batch_id, delay)
```

Comment:
Please change the default value for the `timeout` kwarg on `wait_for_results` instead.

Reply:
https://github.com/redhat-openshift-ecosystem/operator-pipelines/blob/main/operator-pipeline-images/operatorcert/entrypoints/index.py#L44

Looks Good ? @amisstea 

```
New:
line 44:
def wait_for_results(iib_url: str, batch_id: int, timeout=60 * 60, delay=20) -> Any:
line 51:
timeout ([type], optional): Maximum wait time. Defaults to 60*60 (3600 seconds/1 hour).
```

Reply:
That's the correct location. Please push your changes so they can be reviewed.